src/Interpreters/processColumnTransformers.h

```diff
@@ -16,4 +18,10 @@ ASTPtr processColumnTransformers(
         const StorageMetadataPtr & metadata_snapshot,
         ASTPtr query_columns);
 
+ASTPtr processColumnTransformers(
+        const ColumnsDescription & columns,
+        ASTPtr query_columns,
+        const String & current_database,
+        const StorageID & table_id);
```

Comment:
nit: let's reorder the parameters for readability, matching the first function overload:
```
const String & current_database,
const StorageID & table_id,
const ColumnsDescription & columns,
ASTPtr query_columns
```